lib/common/templates/compose/object.kt.template

```diff
@@ -0,0 +1,32 @@
+<%
+//
+// Copyright 2020 Target Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+%>
+<%= fileHeader({file, commentStyle: 'short'}) %>
+
+package <%= file.packageName %>;
+
+<%  // Ideally we would selectively import the correct classes & functions based on whether the applied transforms actually need them. %>
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.*
+
+object <%= file.className %> {
+  <%= allProperties.map(function(prop) {
+    let comment = "";
+    if (prop.comment) {
+      comment = `/** ${prop.comment} */\n  `
+    }
+    return `${comment}val ${prop.name} = ${prop.value}`;
```

Comment:
Here is where you would use the `formatProperty(prop)` function you built in the format function. This is how you could support outputReferences in this format.